### PR TITLE
Update LostorStolenEdenredCard.md

### PR DESCRIPTION
**Pull request by robeecaneda1**
Added steps on how to process lost/stolen cards such as sending an email to Edenred to send the new card to consultant's address and sending a confirmation email to consultant that the new card has been ordered.

### CODE REVIEW

File: Bullhorn/LostorStolenEdenredCard.md [hunk 1]
```diff
@@ -16,6 +16,22 @@ Trigger: Admin receives an email
 - Tick name of employee
 - Hit "Validate my Selection" and confirm
 - Mention on Slack #general who lost his card
+- Send an email to Edenred that the new card should be delivered to the consultant's address
```
Comment (Laoujin) [inline Bullhorn/LostorStolenEdenredCard.md]: Add the email address where it should be sent to

File: Bullhorn/LostorStolenEdenredCard.md [hunk 2]
```diff
@@ -16,6 +16,22 @@ Trigger: Admin receives an email
 - Tick name of employee
```
Comment (Laoujin) [inline Bullhorn/LostorStolenEdenredCard.md]: Missing title

File: Bullhorn/LostorStolenEdenredCard.md [hunk 3]
```diff
@@ -16,6 +16,22 @@ Trigger: Admin receives an email
...
+Een nieuwe Edenred kaart is aangevraagd.
+Deze zal bij u thuis geleverd worden.
+
+Gelieve een bevestiging te sturen naar expenses@itenium.be zodra u de kaart in uw bezig heeft.
+Wij zullen deze dan activeren.
+
+Mvg,
+Team Administratie
+```
+- Email consultant that the new card has been ordered
```
Comment (Laoujin) [inline Bullhorn/LostorStolenEdenredCard.md]: From which email address?

**Comment by Laoujin** [comment]
Not sure if it is necessary to add "Trigger consultant emails to expenses@itenium.be that he has received his card"

+ The actual activation.

File: Bullhorn/LostorStolenEdenredCard.md (hunk 1 above)
Comment (Laoujin) [reply Bullhorn/LostorStolenEdenredCard.md]: This has not yet been resolved.

Comment (robeecaneda1) [reply Bullhorn/LostorStolenEdenredCard.md]: There were conflicts on the PR that I tried to resolved by editing the file the way it should be. Not sure if I did it right or accidentally did something wrong. Not sure if the merge I did to resolve it went directly to master or to Patch-5 (where it should be) Please let me know your thoughts, I can try to revert it if it's wrong. Thanks!

Comment (robeecaneda1) [reply Bullhorn/LostorStolenEdenredCard.md]: For the deleted file, I was able to revert it from github desktop history. Waiting on your feedback on this, quite anxious about the actions I took to make the corrections. Appreciate it.

**Pull request merged**